### DISCUSSION TITLE
What files from NuGet packages are displayed in Visual Studio?

### DISCUSSION THREAD

**Discussion by paulomorgado** [dotnet/project-system#9420] [Q&A]
Expanding a NuGet package in Visual Studio 2022 Solution Explorer shows several files from packages:

![image](https://github.com/dotnet/project-system/assets/470455/4cf8fc58-3838-4275-b9fd-f702598c9cd6)

What are the files shown?

Can additional files or file types be added?

https://github.com/NuGet/Home/discussions/13260

**Comment by drewnoakes** [comment]
The implementation is here: https://github.com/NuGet/NuGet.Client/tree/dev/src/NuGet.Clients/NuGet.VisualStudio.Implementation/SolutionExplorer

What items would you like to see that aren't currently present?

**Reply by paulomorgado** [reply]
Nothing in particular.

I was looking for this to surface .editorconfig files, but could be anything.

That's why I thought this would be primarily a NuGet feature.

I wouldn't mind explicitly declaring on the package reference the files I would like to see under the package reference or have the .props file inside the package declare it.
